markdown.go

```diff
@@ -209,3 +209,10 @@ func (md *MarkdownDoc) githubSourceURL(file string, line int) string {
 	// absolute
 	return fmt.Sprintf("https://%s#L%d", file, line)
 }
+
+func normalizer(s string) string {
```

Comment:
This PR looks good. However, should this '/*' replace logic be directly in chi.Walk()?